requirements.txt

```diff
@@ -1,33 +1,33 @@
-alembic==1.5.4
+alembic==1.13.1
 attrs==21.2.0
 autopep8==1.5.5
 certifi==2020.12.5
 chardet==4.0.0
-click==7.1.2
-Flask==1.1.2
+click==8.1.7
+Flask==3.0.2
 Flask-Cors==3.0.10
-Flask-Migrate==2.6.0
-Flask-SQLAlchemy==2.4.4
+Flask-Migrate==4.0.5
+Flask-SQLAlchemy==3.1.1
 gunicorn==20.1.0
```

Comment:
There is a newer versions of gunicorn (21.2.0) that I've updated hello-books to without issues on deployment, I think we should go with the newer version here too. Do you know if this was updated on the Flasky project or is it using 20.1.0 as well?

Reply:
Let me check on Flasky and make the update here for gunicorn. Thanks!